databricks/sdk/core.py

```diff
@@ -1118,3 +1120,82 @@ def _redacted_dump(self, prefix: str, body: str) -> str:
             return "\n".join([f'{prefix}{line}' for line in json.dumps(raw, indent=2).split("\n")])
         except JSONDecodeError:
             return f'{prefix}[non-JSON document of {len(body)} bytes]'
+
+
+class StreamingResponse(BinaryIO):
+    _response: requests.Response
+    _buffer: bytes
+    _content: Iterator[any]
```

Comment:
I think this should be `Iterator[bytes]`: https://github.com/urllib3/urllib3/blob/1adddfd93ba0f880d74652ca7fe4ca69160ff2d5/src/urllib3/response.py#L381